try{

    var a = 100;

    console.log("El valor de 'a':", a);

}catch(error){

    console.log("Error de catch",error);

}

//si no se utiliza el "catch" tiene que ir el "finally{}"

